yaml/europeandataportal.yaml

```diff
@@ -7,29 +7,30 @@ file_descriptor:
   resources: 
```

Comment:
Question: shouldn't these two yaml files be part of the OPAL project instead of being part of Squirrel?

Reply:
yeah, you are right. I will move it to the opal project